src/ipc/ipc-common/Registry.h

```diff
@@ -29,43 +29,68 @@
 **********************************************************************/
 #pragma once
 
+#include <QObject>
 #include <QString>
 #include <QMap>
 #include <functional>
 #include <QVector>
+#include <QDebug>
+
+struct RegistryListenerEntry {
```

Comment:
just Listener. Less means more. If you afraid names conflict, please namespace it.

Reply:
Bump.

Reply:
Class turned into an inner class of Registry